web-common/src/main/java/edu/unc/lib/boxc/web/common/utils/AnalyticsTrackerUtil.java

```diff
@@ -118,11 +118,10 @@ private void trackEvent(AnalyticsUserData userData, String category, String acti
 
     private MatomoRequest buildMatomoRequest(String url, AnalyticsUserData userData, String parentCollection, String label) throws UnsupportedEncodingException {
         return MatomoRequest.builder()
-                .siteId(MATOMO_SITE_ID)
+                .siteId(matomoSiteID)
                 .visitorId(userData.uid)
                 .actionUrl(urlEncode(url))
-                .actionName(urlEncode(MATOMO_ACTION))
-                .downloadUrl(urlEncode(url))
+                .actionName(urlEncode(parentCollection + " / " + MATOMO_ACTION))
```

Comment:
I remember you mentioning that they suggest a name in the form of "something1 / something2". Is the first something supposed to be the category? I was thinking about whether it'd be more useful recording a value that is the parent collection + download, or the title of the object + download. I'm not really sure one way or the other. Anyways, I'm fine as is, was just trying to remember the expected usage.

Reply:
Yes, the parent collection is the category. This is exactly how it's used in GA so I kept it the same.